lib/Db/AclMapper.php

```diff
@@ -43,8 +45,13 @@ public function isOwner($userId, $aclId): bool {
 	}
 
 	public function findBoardId($aclId): ?int {
-		$entity = $this->find($aclId);
-		return $entity->getBoardId();
+		try {
+			$entity = $this->find($aclId);
+			return $entity->getBoardId();
+		} catch (DoesNotExistException $e) {
+		} catch (MultipleObjectsReturnedException $e) {
```

Comment:
Cant these catch be merged as `catch (DoesNotExistException | MultipleObjectsReturnedException $e) {`?